asv_bench/benchmarks/gil.py

```diff
@@ -1,8 +1,6 @@
 import numpy as np
 import pandas.util.testing as tm
-from pandas import (DataFrame, Series, rolling_median, rolling_mean,
-                    rolling_min, rolling_max, rolling_var, rolling_skew,
-                    rolling_kurt, rolling_std, read_csv, factorize, date_range)
```

Comment:
I might be 'overhead' to keep, but we can also simply move those imports to withing the `else` statement, and then this keeps working for old releases.